### ISSUE TITLE
Ratings are not screen reader accessible

### ISSUE THREAD

**Issue by nerualuhs** [BLVSP-Platform/access-code-hub#53]
<img width="484" height="828" alt="Image" src="https://github.com/user-attachments/assets/098cbd7b-ce48-4f47-81d1-d0b1a48ae7f5" />

ratings are being read as radiogroups and it doesn't read the rating, just the individual stars.

**Comment by fiveminus1** [comment]
This should be fixed now. Silktide may not accurately reflect this, but if you use NVDA, it should read out the aria-label "3 out of 5 stars" or "no rating available."
